src/routes/collabs/[id]/index.svelte

```diff
@@ -98,6 +98,42 @@
 		await getPicks();
 	}
 
+	async function exportcsv() {
+		const collabId = $page.params['id'];
+
+		let res = await fetch('/api/collabs/' + collabId + '/exportcsv');
+
+		// Dont ask me i copyd this from a thread
+		let blob = await res.blob();
+		var url = window.URL || window.webkitURL;
+		let link = url.createObjectURL(blob);
+
+		let a = document.createElement('a');
+		a.setAttribute('download', `export.csv`);
+		a.setAttribute('href', link);
+		document.body.appendChild(a);
+		a.click();
+		document.body.removeChild(a);
+	}
+
+	async function exportassets() {
+		const collabId = $page.params['id'];
+
+		let res = await fetch('/api/collabs/' + collabId + '/exportassets');
+
+		// Dont ask me i copyd this from a thread
+		let blob = await res.blob();
+		var url = window.URL || window.webkitURL;
+		let link = url.createObjectURL(blob);
+
+		let a = document.createElement('a');
+		a.setAttribute('download', `export.zip`);
+		a.setAttribute('href', link);
+		document.body.appendChild(a);
+		a.click();
+		document.body.removeChild(a);
+	}
```

Comment:
Can we merge these two into a single `/api/collabs/{id}/export` call? Perhaps seperate the logic of the two exports in server side functions, and add the CSV into the archive. It doesn't make sense to do have separate downloads for this.

Also, please don't use the DOM lol, using `res.download` in the server and an `<a>` tag in the HTML itself achieves the same thing, just cleaner, without direct DOM manipulation, or the extra javascript. It's fine to extend the `SolidButton` to be able to have children, that'd be much nicer.